test/actions/down.test.js

```diff
@@ -18,11 +18,17 @@ describe("down", () => {
       Promise.resolve([
         {
           fileName: "20160609113224-first_migration.js",
-          appliedAt: new Date()
+          appliedAt: new Date(),
+        },
+        {
+          fileName: "20160609113224-second_migration.js",
```

Comment:
You should change the timestamp of second migration as currently the first ends with `...3224` and the last with `...3225`.

It would be more correct to have the second migration to be in the chronological order as the lib sorts the migration that way. So :
- `...3224` : 1st migration
- `...3225` : 2nd migration
- `...3226` : last migration

Here the tests dont fail only because second is after first in the alphabetical order